nautobot/docs/administration/nautobot-server.md

```diff
@@ -353,9 +353,56 @@ nautobot-server nbshell
 Prompt provided:
 
 ```no-highlight
-### Nautobot interactive shell (localhost)
-### Python 3.8.7 | Django 3.1.7 | Nautobot 1.0.0
-### lsmodels() will show available models. Use help(<model>) for more info.
+# Shell Plus Model Imports
```

Comment:
Is it helpful that Shell Plus prints out the imports on startups instead of the former which was quieter at startups?

Reply:
IMO yes, as it removes the need for us to provide `lsmodels()` as a custom function the way the previous implementation did, since the models are all explicitly shown.

The command does support a `--quiet-load` option which suppresses this output:

```
# nautobot-server nbshell --quiet-load
Python 3.7.13 (default, May 11 2022, 08:57:12) 
[GCC 10.2.1 20210110] on linux
Type "help", "copyright", "credits" or "license" for more information.
(InteractiveConsole)
>>> 
```

I can add a note about this if you think it'd be helpful, though there are a ton of other options to this command that are already well-documented in django-extensions.